source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -626,16 +626,16 @@ bool ContextImpl::verifySubjectAltName(X509* cert,
   return false;
 }
 
-bool ContextImpl::dNSNameMatch(const std::string& dNSName, const char* pattern) {
-  if (dNSName == pattern) {
+bool ContextImpl::dNSNameMatch(const std::string& d_ns_name, const char* pattern) {
```

Comment:
Just a drive-by, but I think this can be `dns_name`. I'm also not sure why the method is `dNSNameMatch` and not `dnsNameMatch` but that can wait for another day.

Reply:
looks like carryover from a funky BoringSSL name: https://github.com/google/boringssl/blob/master/include/openssl/x509v3.h#L197

It might've been intentional to preserve it, but I don't think it adds much value and breaks style uniformity in Envoy so I went ahead and updated it while we're here.